test/dynamics/common.py

```diff
@@ -101,6 +106,32 @@ def jit_grad_wrap(self, func_to_test: Callable) -> Callable:
         return wf(f)
 
 
+class TestDiffraxBase(unittest.TestCase):
+    """Base class with setUpClass and tearDownClass for importing diffrax solvers
+
+    Test cases that inherit from this class will automatically work with diffrax solvers
+    backend.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        try:
+            with warnings.catch_warnings():
+                warnings.simplefilter("ignore")
+                # pylint: disable=import-outside-toplevel
+                import diffrax  # pylint: disable=unused-import
+        except Exception as err:
+            raise unittest.SkipTest("Skipping diffrax tests.") from err
+
+    def Dopri5(self) -> Callable:
```

Comment:
Is this necessary? Can you not just import it in whatever file you're using this in?

Edit: It seems like you're importing them anyway in other files.

Reply:
Yes, I had set this up for an issue with importing, which was solved later.

Reply:
Removed this